0000-ilp-over-http.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ILP Over HTTP
+draft: 1
+---
+
+# ILP Over HTTP
+> A bilateral communication protocol for server-to-server connections
+
+## Motivation
+
+Scaling Interledger infrastructure to handle large volumes of ILP packets requires horizontally scaling connectors. Using HTTP for bilateral communication enables service providers to leverage standard tools and services for hosting, load balancing, Distributed Denial of Service (DDoS) protection, and monitoring.
+
+## Overview
+
+In an ILP Over HTTP connection, both peers run HTTP servers with accessible HTTPS endpoints. When peering, the peers exchange their respective URLs, authentication tokens or TLS certificates, ILP addresses, and settlement-related details.
+
+Each ILP Prepare packet is sent as the body of an HTTP request to the peer's server endpoint. The HTTP response body will contain either the ILP Fulfill or Reject packet.
+
+## Specification
+
+This is a minimal protocol built on HTTP. HTTP/2 is RECOMMENDED for performance reasons, although HTTP/1.1 MAY also be used. Implementations SHOULD support HTTP version negotiation via Application Protocol Negotiation (ALPN).
+
+### Authentication
+
+Peers MAY use any standard HTTP authentication mechanism to authenticate incoming requests. Bearer Tokens or TLS Client Certificates are RECOMMENDED.
+
+### Request
+
+```http
+POST /ilp HTTP/2.0
+Accept: application/octet-string
+Content-Type: application/octet-string
+Authorization: Bearer zxcljvoizuu09wqqpowipoalksdflksjdgxclvkjl0s909asdf
+Host: connector.example
+X-ILP-Destination: example.usd.connector.account
```

Comment:
The `X-` is no longer standard practice: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers "Custom proprietary headers can be added using the 'X-' prefix, but this convention was deprecated in June 2012, because of the inconveniences it caused when non-standard fields became standard in RFC 6648;"